File: Go/Slice/get-last-element-in-slice.go
package main

import (
	"fmt"
)

func main() {
	// Create a slice of strings.
	randomSlice := []string{"A", "B", "C", "D", "E"}
	// Get the last element of the slice.
	fmt.Println(lastElementOfSlice(randomSlice))
}

// Get the last element of a slice.
func lastElementOfSlice(slice []string) string {
	return slice[len(slice)-1]
}

File: Python/Slice/get_last_element_in_slice.py
# Get the last element in the slice.
def get_last_element_in_slice(provided_slice):
    return provided_slice[-1]


def main():
    # Make a slice using a random set of data.
    randomSlice = ["c", "a", "b"]
    # Get the last element in the slice.
    print(get_last_element_in_slice(randomSlice))


main()
